application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -173,30 +178,42 @@ private MessageCreateAction createModMessage(String reportReason, String authorN
             .setColor(AMBIENT_COLOR)
             .build();
         return modMailAuditLog.sendMessageEmbeds(reportedMessageEmbed, reportReasonEmbed)
-            .addActionRow(
-                    DiscordClientAction.General.USER.asLinkButton("Author Profile", authorID));
+            .addActionRow(DiscordClientAction.Channels.GUILD_CHANNEL_MESSAGE.asLinkButton(
+                    "Reported Message", guild.getId(), reportMessage.reportedMessageChannelID,
```

Comment:
maybe its clearer to write `"Go to message"`?

Reply:
Go to reported Message? maybe clearer

Reply:
Il test it out